scout/server/blueprints/variants/templates/variants/str-variants.html

```diff
@@ -64,13 +65,19 @@
 	            <tr>
 	        {% endif %}
             <td>{{ cell_rank(variant) }}</td>
-            <td class="text-right">{{ variant.rank_score|int }}</td>
-            <td>{{ variant.chromosome }}</td>
             <td>{{ variant.str_repid }}</td>
 	          <td class="text-right">{{ variant.str_ru }}</td>
-            <td>{{ variant.alternative }}</td>
+            <td class="text-right">{{ variant.alternative|replace("STR", "")|replace("<", "")|replace(">", "") }}</td>
             <td class="text-right">{{ variant.str_ref }}</td>
             <td>{{ variant.str_status }}</td>
+            <td>{{ variant.chromosome }}</td>
+            <td>
+                {% if case.bam_files %}
+                  <a class="btn btn-default btn-sm" href="{{ url_for('alignviewers.igv', sample=case.sample_names, build=case.genome_build, bam=case.mt_bams, bai=case.mt_bais, contig=variant.chromosome, start=(variant.position - 50), stop=(variant.position + 50), center_guide="T") }}" target="_blank">{{ variant.position}}</a>
```

Comment:
Nice!

Reply:
:blush: There was actually an error there - I apparently tried to load the mt_bams instead of bam_files. Otherwise nice. 😄